tests/vmi_configuration_test.go

```diff
@@ -1671,18 +1673,20 @@ var _ = Describe("[sig-compute]Configurations", func() {
 		})
 
 		Context("[Serial]using defaultRuntimeClass configuration", func() {
-			const runtimeClassName = "fake-runtime-class"
+			var runtimeClassName string
```

Comment:
An alternative is to take the second test case out of `Context` and make it parallel. As 1. The Context doesn't make sense(the case don't use it...) 2. The case doesn't need random runtimeClass...